pkg/helper/versions/helpers.go

```diff
@@ -55,6 +56,13 @@ func GetVersionList(r *rosa.Runtime, channelGroup string, isSTS bool, isHostedCP
 	return
 }
 
+func GetDefaultVersion(version *v1.Version, isHostedCP bool) string {
```

Comment:
```suggestion
func getDefaultVersion(version *v1.Version, isHostedCP bool) string {
```

If it doesn't need to be exposed then let's make it a private helper function.

Reply:
@chenz4027 please feel free to resolve. You have addressed my comment. Thanks.